tests/models/test_taskinstance.py

```diff
@@ -2109,8 +2116,12 @@ def test_execute_queries_count_store_serialized(self):
                 run_type=DagRunType.SCHEDULED,
                 session=session,
             )
-
-        with assert_queries_count(10):
+        # an extra query is fired in RenderedTaskInstanceFields.delete_old_records
+        # for other DBs
+        expected_query_count_based_on_db = (11, 10)[
+            session.bind.dialect.name in ["postgresql", "sqlite", "mysql"]
+        ]
```

Comment:
should use similar logic as above -- looks cleaner

Reply:
fixed in https://github.com/apache/airflow/pull/9973/commits/6c9df7c658f664c8296af887a2796b3ac082649e